### DISCUSSION TITLE
Cypress end-to-end tests break after updating to 13.4.13

### DISCUSSION THREAD

**Discussion by ClementDreptin** [vercel/next.js#56405] [Help]
### Summary

I tried to upgrade from 13.4.12 to 13.4.13 and our Cypress end-to-end tests started to break on our GitLab CI server. Cypress says
```
Cypress could not verify that this server is running:
  > http://localhost:3000/
```
I would have created an issue instead of a discussion if I was able to create a minimal reproduction but the problem only seems to happen on the CI server, running the Cypress tests locally works fine.
I tried to revert back to 13.4.12 just to make sure it wasn't a network config problem on the CI server and Cypress was able to detect the Next.js server, so it seems like something has changed on the Next.js side.

Has anything changed in the Next.js production server in 13.4.13? I found [this PR](https://github.com/vercel/next.js/pull/53174) while looking at the [13.4.13 release notes](https://github.com/vercel/next.js/releases/tag/v13.4.13) but I don't know if it's related.

### Additional information

```js
Output of `next info`

    Operating System:
      Platform: linux
      Arch: x64
      Version: #93~20.04.1-Ubuntu SMP Wed Sep 6 16:15:40 UTC 2023
    Binaries:
      Node: 18.18.0
      npm: 9.8.1
      Yarn: N/A
      pnpm: N/A
    Relevant Packages:
      next: 13.4.13
      eslint-config-next: 13.5.4
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.2.2
    Next.js Config:
      output: standalone
```
Note that we self host the production build so we use the standalone output mode.
```


### Example

_No response_

**Comment by FlyingTurkman** [comment]
I think before the test you should start server with npm run dev

**Comment by ClementDreptin** [comment]
It suprisingly worked but running the tests on the development server instead of the production server is not optimal for memory/cpu usage, and having to create the production build makes sure the code compiles. I just think it's a weird practice to run end-to-end tests on a debug version of the app the users will never see.

**Reply by FlyingTurkman** [reply]
Yes its true. My advice is build it first with npm run build & npm start its much better way
